merkletree/proof.go

```diff
@@ -59,6 +59,10 @@ func (ap *AuthenticationPath) authPathHash() []byte {
 	return hash
 }
 
+func (ap *AuthenticationPath) VerifyBinding(key []byte) bool {
+	return bytes.Equal(ap.Leaf.Value, key)
```

Comment:
I think it depends on how we store the key and the policy into the binding. That should be addressed in #46. Thus, the client has to know the storage schema. Another question is do we expect this?